src/worker.jl

```diff
@@ -200,21 +221,70 @@ function worker_init(f::File)
             return take!(buf)
         end
 
-        function render_mimetypes(value)
+        # TODO: where is this key?
+        function _to_format()
+            fm = FRONTMATTER[]
+            if haskey(fm, "pandoc")
+                pandoc = fm["pandoc"]
+                if isa(pandoc, Dict) && haskey(pandoc, "to")
+                    to = pandoc["to"]
+                    isa(to, String) && return to
+                end
+            end
+            return nothing
+        end
+
+        function render_mimetypes(value, cell_options)
+            to_format = _to_format()
             result = Dict{String,@NamedTuple{error::Bool, data::Vector{UInt8}}}()
-            mimes = [
-                "text/plain",
-                "text/html",
-                "text/latex",
-                "image/svg+xml",
-                "image/png",
-                "application/json",
-            ]
+            # Some output formats that we want to write to need different
+            # handling of valid MIME types. Currently `docx` and `typst`. When
+            # we detect that the `to` format is one of these then we select a
+            # different set of MIME types to try and render the value as. The
+            # `QuartoNotebookRunner/*` MIME types are unique to this package and
+            # are how package authors can hook into the display system used here
+            # to allow their types to be rendered correctly in different
+            # outputs.
+            #
+            # NOTE: We may revise this approach at any point in time and these
+            # should be considered implementation details until officially
+            # documented.
+            mime_groups = Dict(
+                "docx" => [
+                    "QuartoNotebookRunner/openxml",
+                    "text/plain",
+                    "text/latex",
+                    "text/html",
```

Comment:
not sure that html makes sense for docx, I mean they do some postprocessing on html for example to extract tables, but still.

Reply:
Yeah, hadn't tested any docx behaviour, we may be able to just discard that mime from the docx stack.